test/integration/contractor_ends_project_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.